contentcuration/contentcuration/viewsets/websockets/consumers.py

```diff
@@ -128,3 +128,12 @@ def receive(self, text_data):
         self.send(json.dumps({
             'response_payload': response_payload
         }))
+
+    # Receive message from room group
```

Comment:
Perhaps move this into the method doc block. Also, might be good to expand on this comment-- if I'm understanding everything correctly, perhaps something like:
```
"""
Receive message events sent to the subscribed groups from our Django signal handlers, and relay the messages to the frontend
"""
```